tests/metrics/classification/test_accuracy.py

```diff
@@ -3,134 +3,92 @@
 import torch
 import numpy as np
 from collections import namedtuple
+from functools import partial
 
 from pytorch_lightning.metrics.classification.accuracy import Accuracy
 from sklearn.metrics import accuracy_score
 
-from tests.metrics.utils import compute_batch, NUM_BATCHES, BATCH_SIZE
+from tests.metrics.utils import compute_batch, setup_ddp
+from tests.metrics.utils import THRESHOLD
+
+from tests.metrics.classification.utils import (
```

Comment:
nice 👍 
maybe include in the function name that these functions are meant for testing
like `_testing_inputs_binary_prob`